jdk/test/javax/swing/JFrame/8016356/bug8016356.java

```diff
@@ -21,12 +21,17 @@
  * questions.
  */
 
-/* @test
-   @bug 8016356
-   @summary Any swing frame resizes ugly.
-   @run main bug8016356
-   @author Oleg Pekhovskiy
-*/
+/**
+ * @test
+ * @key headful
+ * @bug 8016356
+ * @summary Any swing frame resizes ugly.
+ * @author Oleg Pekhovskiy
+ * @requires (os.family == "windows")
+ * @library ../../../../lib/testlibrary
+ * @build jdk.testlibrary.OSInfo
```

Comment:
do we need all these new things to add?

Reply:
Add new things test passed locally after deleting '* @requires (os.family == "windows")'.

Reply:
Why did you add it?

Reply:
https://bugs.openjdk.org/browse/JDK-8185500 add '@requires (os.family == "windows")', so I also add it when backport JDK-8185500.

Reply:
But you also added this
 * @library ../../../../lib/testlibrary
 * @build jdk.testlibrary.OSInfo

Reply:
I see. "@library ../../../../lib/testlibrary" and "@build jdk.testlibrary.OSInfo" were add by https://bugs.openjdk.org/browse/JDK-8058726, for this time backport is not necessary. I will delete it.